index.rst

```diff
@@ -6,24 +6,474 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
+.. Uncomment to use citations
+.. .. rubric:: References
+.. 
+.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+..    :style: lsst_aa
 
 Abstract
 ========
 
-Principles, guidelines, conventions, formatting style, and workflow for tutorials developed by the Community Engagement Team (CET).
+A guideline for anyone creating or updating tutorials for the Rubin science community.
+This includes all community-facing demonstrations of how to use the Rubin Science Platform or how to analyze LSST data.
+
+Pedagogical principles
+======================
+
+All tutorial developers should adhere to the following principles.
+
+Inclusivity
+-----------
+
+Avoid offensive or exclusionary language at all times (e.g., violent or ableist terms).
+Ensure jargon and acronyms are defined.
+
+Level-appropriate
+-----------------
+
+Tutorials should clearly identify and teach to their target audience (beginner, intermediate, or advanced).
+
+Skill-focused
+-------------
+
+Tutorials should focus on teaching one or a few new skills or techniques, or providing one scientific demonstration.
+
+Consumable
+----------
+
+The user should be able to work through the core contents of a tutorial within 30 minutes.
+Additional examples could be included, but should be clearly labeled as extra steps.
+
+Well-documented
+---------------
+
+High quality documentation should be provided with the tutorial, including narrative descriptions, citations, references, 
+and external links to, e.g., package documentation.
+
+Cross-referenced
+----------------
+
+Tutorials should reference an precursor or advanced tutorials that users should consider as pre-requisit or follow-up resources.
+
+Properly credited
+-----------------
+
+Appropriate acknowledgements should be provided, in order to give credit to individuals whose notebooks were used as examples 
+and to set a precedent of prioritizing credits in an openly collaborative environment.
+
+
+Jupyter notebook format, style and code standards
+=================================================
+
+Template
+--------
+
+Use the `template notebook <https://github.com/rubin-dp0/cet-dev/blob/main/template.ipynb>`_ in the rubin-dp0 GitHub 
+Organization's ``cet-dev`` repository as a starting point.
+The template contains an example of the header and the mandatory first section described below.
+
+Header
+^^^^^^
+
+The structure of the header is mandatory.
```

Comment:
Perhaps these and further instructions would benefit from screenshots of what is being referred to.

Reply:
Going to check with Melissa on this comment and add to the next version.